client/reader/search-stream/style.scss

```diff
@@ -373,6 +373,7 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 
 .search-stream__results.is-two-columns .search-stream__post-results {
 	max-width: 660px;
```

Comment:
I think we don't need max-width anymore since we've set a fixed value for width.